audio-worklet/design-pattern/awn-shared-buffer/shared-buffer-worklet-processor.js

```diff
@@ -0,0 +1,145 @@
+// In this design, this AudioWorkletProcessor simply functions as a audio
+// callback (or audio sink). The actual audio processing happens on the Worker
+// side.
+
+// Description of shared states.
+const STATE = {
```

Comment:
Presumably this has to be consistent with STATE above.  Is there any way to define this in just one place so that we're guaranteed that they're consistent.